code/game/objects/items/tools/multitool.dm

```diff
@@ -105,6 +105,33 @@
 	toolspeed = 0.95 // dangerously fast... not like multitools use speed anyways
 	origin_tech = "magnets=1;engineering=2;syndicate=1"
 
+/obj/item/multitool/command
+	name = "command multitool"
+	desc = "Used for pulsing wires to test which to cut. Not recommended by the Captain."
+	icon_state = "multitool_command"
+	belt_icon = "multitool_command"
+	toolspeed = 0.95 //command those wires / that fireaxe cabinet!
+
+/obj/item/multitool/command/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] is attempting to command the command multitool! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+    //basically just cleaned up and copied from the medical wrench code
+	if(!user)
+		return
+
+	user.Stun(10 SECONDS)
+	playsound(loc, 'sound/effects/supermatter.ogg', 50, 1, -1)
+	sleep(20)
+
+	for(var/obj/item/W in user)
+		user.unEquip(W)
+
+	var/obj/item/multitool/command/M = new /obj/item/multitool/command(loc)
```

Comment:
Actually, won't this drop the original and make a second one?

Reply:
I'll be honest, I basically copy-pasted this from medical wrench code. I'll take a look in a minute and double-check to make sure this doesn't happen